py3status/modules/imap.py

```diff
@@ -34,15 +33,27 @@ class Py3status:
     cache_timeout = 60
     criterion = 'UNSEEN'
     format = 'Mail: {unseen}'
+    # remove this? // use format = '[\?not_zero Mail: {unseen}]'
     hide_if_zero = False
     imap_server = '<IMAP_SERVER>'
     mailbox = 'INBOX'
-    new_mail_color = ''
+    msg_unavailable = 'Mail: N/A'
     password = '<PASSWORD>'
     port = '993'
     security = 'ssl'
     user = '<USERNAME>'
 
+    class Meta:
+        deprecated = {
+            'rename': [
+                {
+                    'param': 'new_mail_color',
+                    'new': 'color_good',
```

Comment:
`color_new_mail` would be nicer

This will need testing as I'm worried we may not pass this through, but that would be something I'd fix but good to know if it's an issue.